lib/Catalyst/Authentication/Credential/Twitter.pm

```diff
@@ -46,67 +46,69 @@ sub new {
             Catalyst::Exception->throw("$param not defined") 
     }
 
-    # Create a Net::Twitter instance
-    $self->_twitter(Net::Twitter->new({ 
-		'traits'        	=> ['API::RESTv1_1', 'OAuth'],
-		'consumer_key' 		=> $self->consumer_key, 
-        'consumer_secret'	=> $self->consumer_secret,
-		'ssl'				=> 1,
-	}));
+    # Create a Twitter::API instance
+    $self->_twitter(Twitter::API->new_with_traits( 'traits' => ['Migration', 'ApiMethods','RetryOnError'],
```

Comment:
The Migration is meant to be a temporary trait for transitioning from Net::Twitter to Twitter::API. It would be better to exclude the Migration trait and call Twitter::API's `oauth_request_token` followed by `oauth_authentication_url`, then `oauth_access_token` in the Twitter callback handler.

Net::Twitter combined getting the request token and the authentication URL in a single call, `get_authentication_url`, which I deemed a design flaw and is one of the very few compatibility breaking changes between Net::Twitter and Twitter::API.

There's an [example of Twitter::API's OAuth flow](https://github.com/semifor/Twitter-API/blob/master/examples/oauth_web.psgi) in the distribution.

(I made this comment on the commit, before I saw the PR. This is a more appropriate place for it, so apologies for the duplication.)